packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -700,14 +701,14 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             (id: string, createParam: CreateChildSummarizerNodeParam) => (
                     summarizeInternal: SummarizeInternalFn,
                     getGCDataFn: () => Promise<IGCData>,
-                    getInitialGCDataFn: () => Promise<IGCData | undefined>,
+                    getInitialGCDetailsFn: () => Promise<IGCDetails | undefined>,
```

Comment:
Maybe Gc (i.e. IGcDetails)? Or full name (IGarbageCollectionDetails)?
GCDetails is hard to read, and it has GCD - pretty often used term in software engineering (Grand Central Dispatch)

Reply:
Didn't know about GCD :-). Will rename it.

Reply:
Personally I like `IGcDetails` but will be inconsistent with other places right now.

Reply:
Renamed to IGarbageCollectionDetails